src/main/steps/applicant1Sequence.ts

```diff
@@ -92,6 +96,7 @@ export const applicant1Sequence: Step[] = [
   {
     url: HAS_RELATIONSHIP_BROKEN_URL,
     showInSection: Sections.AboutPartnership,
+    showInCompleteSection: Sections.AboutPartnership,
```

Comment:
as mentioned above, with the condition i suggested, we dont need to duplicate these, just ones that are different.

Reply:
There are some steps which have Sections assigned that we don't want to show up in the completeSections, so I think we'll need this to differentiate between them